module/input_handler.py

```diff
@@ -98,7 +98,7 @@ def handle_nonglobal_commands(self, user_word, user_command_list) -> bool:
             if correct_params:
                 print("\tSending command:", user_word, user_params, self.cli_controller.target)
                 # below function can currently not be called as there is no string packing support in python bus yet
-                # cast_and_send_ui_frame(self.cli_controller.comm, user_word, user_params, self.cli_controller.selected)
+                cast_and_send_ui_frame(self.cli_controller.comm, user_word, user_params, self.cli_controller.target)
```

Comment:
this should stay commented until a fix has been implemented for [this issue](https://github.com/R2D2-2019/r2d2-python-build/issues/47) by python build